scripts/run_installer.sh

```diff
@@ -18,20 +18,24 @@
 set -o errexit
 set -o nounset
 
-GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
-GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
+_GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
+# Environment variable file should be stored in the same directory as
+# this script.
+_GPU_INSTALLER_ENV_PATH="$(dirname "$0")/gpu-installer-env"
+
 # The following environment variables may be changed by cos-gpu-installer-env.
 COS_NVIDIA_INSTALLER_CONTAINER="gcr.io/cos-cloud/cos-gpu-installer:latest"
 NVIDIA_INSTALL_DIR_HOST="/var/lib/nvidia"
 NVIDIA_INSTALL_DIR_CONTAINER="/usr/local/nvidia"
 ROOT_MOUNT_DIR="/root"
 
 setup() {
-  if [ ! -f "${GPU_INSTALLER_ENV_PATH}" ]; then
-    /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" \
-      > "${GPU_INSTALLER_ENV_PATH}" || true
+  local env_from_metadata=$(/usr/share/google/get_metadata_value \
```

Comment:
nit: if possible, doing this the same way as its done in the cloud-config will be better (i.e. download under /tmp/ and then copy if its successful.
```
  if /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" > /tmp/cos-gpu-installer-env ; then
    cp -f /tmp/cos-gpu-installer-env "${_GPU_INSTALLER_ENV_PATH}"
  fi
```

Reply:
Done.